### ISSUE TITLE
ImportError: cannot import name 'Timeout' from 'mitmproxy.exceptions' 

### ISSUE THREAD

**Issue by tsoernes** [wkeeling/selenium-wire#355]
```
pip list | grep selenium-wire
selenium-wire                      4.3.3
pip list | grep mitmproxy
mitmproxy                          7.0.0
```

```
File "/home/torstein/anaconda3/lib/python3.8/site-packages/seleniumwire/webdriver.py", line 99, in __init__
    self.proxy = backend.create(port=seleniumwire_options.get('port', 0), options=seleniumwire_options)
File "/home/torstein/anaconda3/lib/python3.8/site-packages/seleniumwire/backend.py", line 38, in create
    from . import mitmproxy
File "/home/torstein/anaconda3/lib/python3.8/site-packages/seleniumwire/mitmproxy.py", line 23, in <module>
    from mitmproxy.exceptions import Timeout  # isort:skip

ImportError: cannot import name 'Timeout' from 'mitmproxy.exceptions' (/home/torstein/anaconda3/lib/python3.8/site-packages/mitmproxy/exceptions.py)
```

**Comment by tsoernes** [comment]
`pip install 'mitmproxy<7.0.0'` fixes this problem

**Comment by wkeeling** [comment]
This is happening due to incompatible APIs in the latest version of mitmproxy 7.0.0. 

You should be able to fix by installing version 5.0.3 specifically:

```
pip install mitmproxy==5.0.3
```

The mitmproxy backend functionality is going to be removed fairly soon, but I'll make a patch release to fix this in the meanwhile.
